include/DataStructures.h

```diff
@@ -15,7 +15,7 @@ namespace CoolProp {
 
 struct SimpleState
 {
-    double rhomolar, T, p, hmolar, smolar, umolar, Q;
+    double rhomolar{}, T{}, p{}, hmolar{}, smolar{}, umolar{}, Q{};
```

Comment:
There seems to be some problem with the examples. I have not looked at this in detail, yet.
https://github.com/CoolProp/CoolProp/actions/runs/3735565861/jobs/6592680936#step:8:145

Reply:
I think this just forces default initialization, no?